Islandora-CLAW/Crayfish
# Add YAML Config Files

## GitHub Issue

Part of: 
https://github.com/Islandora-CLAW/CLAW/issues/603

Related to:
https://github.com/Islandora-CLAW/Crayfish-Commons/pull/8

## What does this Pull Request do?

Updates all the micro services to use YAML configuration files, instead of the PHP ones we were using before.

## How should this be tested?

- Setup all the microservices using this PR and https://github.com/Islandora-CLAW/Crayfish-Commons/pull/8
- Test the settings in the configuration file and make sure they all work as they did before when the config files were PHP.

# Interested parties
@Islandora-CLAW/committers
